src/test/java/com/scmspain/karyon/accesslog/AccessLogIntegrationTest.java

```diff
@@ -27,9 +31,10 @@
 public class AccessLogIntegrationTest {
   private static KaryonServer server;
   private static AccessLogFormatter mockedFormatter;
+  private static final  String IP_FOR_TEST = "8.8.8.8";
 
-  @BeforeClass
-  public static void setUpBefore() throws Exception {
+  @Before
+  public void setUp() throws Exception {
```

Comment:
Changing from BeforeClass to Before cause the server is shut down and started up for every tests case, in despite this may means more accurate result, it slows down the performance. Did you change it for any particular reason=?